graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlWSSubProtocol.java

```diff
@@ -352,7 +353,7 @@ private static void writeNext(WebSocketWriter out, String operationId, Execution
                 "payload", executionResult.toSpecification());
         final String event = serializeToJson(response);
         logger.trace("NEXT: {}", event);
-        out.tryWrite(event);
```

Comment:
Several `tryWrite` are still used in this class. Could you also fix it?

Reply:
btw, do i have to handle for `ClosedStreamException` in other place?